src/blinding.cpp

```diff
@@ -583,6 +604,33 @@ LIBSESSION_C_API bool session_blind25_sign(
     }
 }
 
+LIBSESSION_C_API bool session_blind_version_sign_request(
+        const unsigned char* ed25519_seckey,
+        size_t timestamp,
+        const unsigned char* method,
+        size_t method_len,
```

Comment:
Seems like we don't need the length of method, path, and body, can we clean them up?

Reply:
We could remove them, but there are a couple reasons I think it's better to have them.  Without the size parameter, the string must be null-terminated (which is likely to be the case, but I think there's no harm in guarding against it not being the case) and most string-y containers in most languages already know their length, so computing it here (the constructor for the string view would do so) is redundant.

My experience is that C APIs which take an arbitrary length of data tend to take a pointer to the beginning and a length, but if it is decided that having the length parameters is cumbersome I'm more than willing to change it.

Reply:
In general, for things like method and path (which are text, i.e. non-binary, strings) we've stuck with using `const char*` (and no size) in the C API wrappers, with `unsigned char*` used specifically for known-fixed size binary data (such as seckey) and variable size binary data (such as body).  Null-terminated strings are generally extremely easy to wrap (more so than an `unsigned char*, size_t` pair), and so this also signals to the API wrapper that that isn't binary, but is rather a plain text string value.

Reply:
Amended method and path to use `string_view`, leaving body as `ustring_view` as per Jason's recommendation.  C API mirrors that change.